doc/articles/common-issues-vs2022.md

```diff
@@ -67,11 +67,6 @@ Event handlers [cannot be automatically](https://github.com/unoplatform/uno/issu
 
 A workaround is to use the [`x:Bind` to events feature](features/windows-ui-xaml-xbind.md#examples). This feature allows to use a simpler syntax like `<Button Click="{x:Bind MyClick}" />` and declare a simple method `private void MyClick() { }` in the code-behind.
 
-### Abnormally long build times with using WebAssembly and WSL
```

Comment:
@jeromelaban it seems as part of the re-org of the docs that "get-started-with-linux.md" has been removed with the content placed in other files. I couldn't see the content for downgrading to WSL 1 - is this still an appropriate fix? If so, need to add this section back with appropriate link

Reply:
There's no need to downgrade to WSL 1 anymore, and we're not suggesting to use WSL explicitly, only in very specific legacy cases.